troposphere/static/js/components/images/detail/stats/SparklineGraph.jsx

```diff
@@ -33,9 +32,9 @@ export default React.createClass({
         var height = 15;
         var width = 135;
 
-        var plotLines = [],
-            plotBands = [];
-        var chart = new Highcharts.createChart(el, {
+        // FIXME - evaluate why this component doesn't use `chart` anywhere
+        // var chart = ...
+        new Highcharts.createChart(el, {
```

Comment:
So I think you can do the object creation, and just skip assigning to a variable.